MH-13037 Remove unused External API roles

staubesv opened the pull request.

staubesv committed:
MH-13037 Remove unused External API roles

staubesv committed:
MH-13037 Remove unused External API roles from LDAP example configuration

gregorydlogan commented on etc/security/mh_default_org.xml:
```diff
@@ -167,10 +167,6 @@
-    <sec:intercept-url pattern="/api/events/*/attachments" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_ATTACHMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/attachments/*" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_ATTACHMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/comments" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_COMMENTS_VIEW"/>
-    <sec:intercept-url pattern="/api/events/*/comments/*" method="GET" access="ROLE_ADMIN, ROLE_API_EVENTS_COMMENTS_VIEW"/>
```
ROLE_API_EVENTS_COMMENTS_VIEW (at least) is still present in `docs/guides/developer/docs/api/groups-api.md`.  I haven't searched the others, but I would imagine this is true of some of them.

staubesv replied:
Fixed.

I've searched the entire codebase for occurrences of the removed roles now. The one you have found was the only one.

staubesv committed:
MH-13037 Remove obsolete roles from response examples

The External API specification contains example responses. To avoid confusion, the roles removed by MH-13037 have also been removed from that example responses and replaced by obvious example roles to avoid the necessity for further cleanups.
